src/QueryRenderer.ts

```diff
@@ -359,6 +359,8 @@ class QueryRenderChild extends MarkdownRenderChild {
                 // to make a difference.
                 // The issue is tracked here: https://github.com/obsidian-tasks-group/obsidian-tasks/issues/1879
                 await leaf.openFile(file, { eState: { line: line } });
+                // Suppress the default behavior of the event in this case.
```

Comment:
To help future maintainers, it would be really nice if the comment could explain why - and also what is it about 'this case'?

Reply:
With "this case" I meant the case that the task was found. I have made it now more clear. And also removed the outdated comment below (the mentioned issue there has been fixed meanwhile).